torchaudio/utils/sox_utils.py

```diff
@@ -26,10 +26,10 @@ def set_verbosity(verbosity: int):
 
     Args:
         verbosity (int): Set verbosity level of libsox.
-            1: failure messages
-            2: warnings
-            3: details of processing
-            4-6: increasing levels of debug messages
+            1. failure messages
```

Comment:
Thanks for the feedback. I am unable to get the points into a list even after making this change. Any advice?.
![image](https://user-images.githubusercontent.com/4648995/99061165-6cc8e680-25a1-11eb-8b84-aa62ccba4a65.png)

Reply:
@krishnakalyan3 

What about adding hyphen in front?

```
        - 1: failure
        - 2: details of processing
...
```

Reply:
Hi @krishnakalyan3 

I looked at your issue. I tried it on my end and things worked fine. Maybe it's about the browser cache?
If you still have the problem, you can apply the following patch and move on. (Note that there must be one blank line before the start of the list. 

```diff
diff --git a/torchaudio/utils/sox_utils.py b/torchaudio/utils/sox_utils.py
index 076e266..3f9ad0b 100644
--- a/torchaudio/utils/sox_utils.py
+++ b/torchaudio/utils/sox_utils.py
@@ -26,10 +26,11 @@ def set_verbosity(verbosity: int):

     Args:
         verbosity (int): Set verbosity level of libsox.
-            1. failure messages
-            2. warnings
-            3. details of processing
-            4-6. increasing levels of debug messages
+
+            * ``1`` failure messages
+            * ``2`` warnings
+            * ``3`` details of processing
+            * ``4``-``6`` increasing levels of debug messages

     See Also:
         http://sox.sourceforge.net/sox.html
```